spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -158,19 +158,19 @@ def after_hooks(source)
             end
           end
 
-          expect( formatter.legacy_messages ).to eq [
-              :before_features,
-                :before_feature,
+          expect(formatter.legacy_messages).to eq [
+            :before_features,
+              :before_feature,
+                :before_tags,
+                :after_tags,
+                :feature_name,
+                :before_feature_element,
                   :before_tags,
-                  :after_tags,
-                  :feature_name,
-                  :before_feature_element,
-                    :before_tags,
-                      :after_tags,
-                    :scenario_name,
-                  :after_feature_element,
-                :after_feature,
-              :after_features,
+                    :after_tags,
+                  :scenario_name,
+                :after_feature_element,
+              :after_feature,
+            :after_features,
           ]
```

Comment:
Yes, this indentation is there to make the tests easier to read / write, since you need matching pairs of those messages. I'd be happy to see it disappear though if it's a pain - I hope we'll be able to [remove this entire file soon](https://github.com/cucumber/cucumber-ruby/issues/839)